### PR TITLE
Fix handle_response method body key

### PR DESCRIPTION
**Pull request by azimcan**
The erroneous return values ​​of some queries are "error" and some are "errors". I created a temporary solution to this situation. Then the server-side solution can be developed.

### CODE REVIEW

File: lib/hipcall/resource.rb [hunk 1]
```diff
@@ -11,25 +11,25 @@ def initialize(client)
...
 		end
-	
+
 		def delete_request(url, params: {}, headers: {})
 			handle_response client.connection.delete(url, params, headers)
 		end
 
 		def handle_response(response)
-			message = response.body["errors"]
+			message = response.body["error"] || response.body["errors"]
```
Comment (onurozgurozkan) [inline lib/hipcall/resource.rb:32]: @azimcan Which endpoint do you get `error` and which endpoint do you get `errors`?

Comment (azimcan) [reply lib/hipcall/resource.rb:32]: Example
Screenshot 2024-07-19 at 17 57 51

Comment (onurozgurozkan) [reply lib/hipcall/resource.rb:32]: But this is return `errors:` not `error:` . I don't understand what we should change line 32.

```ruby
message = response.body["errors"]
message = response.body["error"] || response.body["errors"]
````

We can fix the "Internal Server Error" but I don't think we should need to add `response.body["error"] ||` at line 32.

**Comment by onurozgurozkan** [comment]
Dear @azimcan ,

I changed all `error` to `errors` from server side. We don't need the change the Hipcall's Ruby SDK.

I'll close this PR.

**Pull request closed without merging**